### ISSUE TITLE
Importing Bootstrap's CSS is not working.

### ISSUE THREAD

**Issue by rjc3rd** [twbs/bootstrap#37138]
Following the documentation on Bootstrap 5.2 Docs, fail to import the files as needed. The failure happens on Parcel, Webpack and Vite. I'm using Visual Studio Code.

https://getbootstrap.com/docs/5.2/getting-started/parcel/
https://getbootstrap.com/docs/5.2/getting-started/webpack/
https://getbootstrap.com/docs/5.2/getting-started/vite/

// Import all of Bootstrap's CSS
@import "~bootstrap/scss/bootstrap";

I was able to make it work by pointing directly to the bootstrap.scss but the documentation seems to be incorrect.

// Import all of Bootstrap's CSS
@import "node_modules/bootstrap/scss/bootstrap.scss";

I'm not sure if my fix was just a workaround, or is the documentation incorrect?
But one thing for sure, following the directions on the GetBootstrap.com website do not work.

**Comment by mdo** [comment]
They do indeed work—we've tested them ourselves and have examples built on them over at https://github.com/twbs/examples as well. Something else is likely off in your setup or dev environment.

Please ask around on the Discussions tab if you need further assistance setting those up, or the specific JS framework's repo if you're having trouble with one.

**Comment by rjc3rd** [comment]
I'd request that you actually just follow the documentation "exactly" as it is written from a new empty folder in Visual Studio Code. I'm sure that your example is working fine but it was built at a different time. As I said in my original post. It probably is supposed to work as it is written, but it doesn't work now.

Noting is off in my setup or dev environment. I was able to solve the problem by changing the syntax,  but ...

Import Bootstrap

Importing Bootstrap into Parcel requires two imports, one into our styles.scss and one into our main.js.

    Import Bootstrap’s CSS. Add the following to src/scss/styles.scss to import all of Bootstrap’s source Sass.

// Import all of Bootstrap's CSS
@import "~bootstrap/scss/bootstrap";

The Documentation Does Not Work.


Changing "~bootstrap/scss/bootstrap"; to "node_modules/bootstrap/scss/bootstrap.scss";

That makes it work. I googled this and found the solution. I don't remember where I found it, and it was suggested that it might be a workaround. But for sure it does NOT work as per the Documentation currently.

**Comment by mdo** [comment]
I just did it again all from scratch, and it works.

<img width="1054" alt="Screen Shot 2022-09-13 at 11 09 37 AM" src="https://user-images.githubusercontent.com/98681/189978136-4359bdb6-127b-4d33-b90a-397c74ed4424.png">

<img width="1030" alt="Screen Shot 2022-09-13 at 11 10 05 AM" src="https://user-images.githubusercontent.com/98681/189978227-217a344d-7cc3-47d7-a2d9-4e89b5a72e5f.png">

<img width="1030" alt="Screen Shot 2022-09-13 at 11 10 02 AM" src="https://user-images.githubusercontent.com/98681/189978241-b1c3baa7-54a6-4e03-9beb-865e12e1b8cd.png">

<img width="1030" alt="Screen Shot 2022-09-13 at 11 10 09 AM" src="https://user-images.githubusercontent.com/98681/189978261-bfded6be-a867-4a1d-a3f0-5715f0cf4ea6.png">


<img width="1013" alt="Screen Shot 2022-09-13 at 11 08 18 AM" src="https://user-images.githubusercontent.com/98681/189978120-d0a731d3-2041-4615-a235-8a929cd97376.png">

**Comment by rjc3rd** [comment]
I appreciate that you did that. Thank you. 

Not sure if I will look any to deeper "solve" this, even though it is not working like that on my end. 

If others have the same issue as me and follow up, that would be awesome.

For now I consider this to be closed. Thank you.

**Comment by ffoodd** [comment]
Looks like you're missing something to resolve node_modules path. I don't know where it should be done in your current configuration but Sass handles a `load-path=node_modules` CLI argument. This is something you can specify in a Webpack config too.

Since your workaround simply uses the full path to node_modules, I bet that's the reason.

Maybe have a look at our exemple projets, if you ever notice something?

**Comment by rjc3rd** [comment]
I concede.

There must have been something wrong in my dev environment.

I started looking into the load-path=node_modules comment and figured out the problem was most likely in my nodejs/npm environment. 

Ultimately after wasting over an hour trying to "fix" it.
I simply sudo apt --purge nodejs and sudo apt install nodejs.

Just like magic. It's works exactly like the documentation says it will.

I followed up solely for anyone else who ever has this problem.
